diff --exclude-from=/home/dang/.scripts/diffrc -up -ruN kvm-79.orig/qemu/net.c kvm-79/qemu/net.c
--- kvm-79.orig/qemu/net.c	2008-11-12 06:48:01.000000000 -0500
+++ kvm-79/qemu/net.c	2008-11-24 21:40:51.000000000 -0500
@@ -133,8 +133,8 @@
 
 #include "qemu-kvm.h"
 
-#define DEFAULT_NETWORK_SCRIPT "/etc/qemu-ifup"
-#define DEFAULT_NETWORK_DOWN_SCRIPT "/etc/qemu-ifdown"
+#define DEFAULT_NETWORK_SCRIPT "/etc/kvm/kvm-ifup"
+#define DEFAULT_NETWORK_DOWN_SCRIPT "/etc/kvm/kvm-ifdown"
 #ifdef __sun__
 #define SMBD_COMMAND "/usr/sfw/sbin/smbd"
 #else
diff --exclude-from=/home/dang/.scripts/diffrc -up -ruN kvm-79.orig/qemu/vl.c kvm-79/qemu/vl.c
--- kvm-79.orig/qemu/vl.c	2008-11-12 06:48:01.000000000 -0500
+++ kvm-79/qemu/vl.c	2008-11-24 21:39:58.000000000 -0500
@@ -152,8 +152,8 @@
 
 #include "qemu-kvm.h"
 
-#define DEFAULT_NETWORK_SCRIPT "/etc/qemu-ifup"
-#define DEFAULT_NETWORK_DOWN_SCRIPT "/etc/qemu-ifdown"
+#define DEFAULT_NETWORK_SCRIPT "/etc/kvm/kvm-ifup"
+#define DEFAULT_NETWORK_DOWN_SCRIPT "/etc/kvm/kvm-ifdown"
 #ifdef __sun__
 #define SMBD_COMMAND "/usr/sfw/sbin/smbd"
 #else
@@ -5200,7 +5200,7 @@ static void help(int exitcode)
            "\n"
            "When using -nographic, press 'ctrl-a h' to get some help.\n"
            ,
-           "qemu",
+           "kvm",
            DEFAULT_RAM_SIZE,
 #ifndef _WIN32
            DEFAULT_NETWORK_SCRIPT,
