diff --exclude-from=/home/dang/.scripts/diffrc -up -ruN kvm-82.orig/qemu/net.h kvm-82/qemu/net.h
--- kvm-82.orig/qemu/net.h	2008-12-24 09:24:58.000000000 -0500
+++ kvm-82/qemu/net.h	2009-01-04 20:10:12.000000000 -0500
@@ -94,8 +94,8 @@ void net_cleanup(void);
 int slirp_is_inited(void);
 void net_client_check(void);
 
-#define DEFAULT_NETWORK_SCRIPT "/etc/qemu-ifup"
-#define DEFAULT_NETWORK_DOWN_SCRIPT "/etc/qemu-ifdown"
+#define DEFAULT_NETWORK_SCRIPT "/etc/kvm/kvm-ifup"
+#define DEFAULT_NETWORK_DOWN_SCRIPT "/etc/kvm/kvm-ifdown"
 #ifdef __sun__
 #define SMBD_COMMAND "/usr/sfw/sbin/smbd"
 #else
diff --exclude-from=/home/dang/.scripts/diffrc -up -ruN kvm-82.orig/qemu/vl.c kvm-82/qemu/vl.c
--- kvm-82.orig/qemu/vl.c	2008-12-24 09:24:58.000000000 -0500
+++ kvm-82/qemu/vl.c	2009-01-04 20:05:23.000000000 -0500
@@ -4113,7 +4113,7 @@ static void help(int exitcode)
            "\n"
            "When using -nographic, press 'ctrl-a h' to get some help.\n"
            ,
-           "qemu",
+           "kvm",
            DEFAULT_RAM_SIZE,
 #ifndef _WIN32
            DEFAULT_NETWORK_SCRIPT,
