diff --exclude-from=/home/dang/.scripts/diffrc -up -ruN kvm-79.orig/scripts/qemu-ifdown kvm-79/scripts/qemu-ifdown
--- kvm-79.orig/scripts/qemu-ifdown	1969-12-31 19:00:00.000000000 -0500
+++ kvm-79/scripts/qemu-ifdown	2008-12-10 11:04:22.000000000 -0500
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+if [ -x /sbin/brctl ]; then
+  BRCTL="/sbin/brctl"
+elif [ -x /usr/sbin/brctl ]; then
+  BRCTL="/usr/sbin/brctl"
+else
+  echo "no bridge utils installed"
+  exit 1
+fi
+
+if [ -x /sbin/ip ]; then
+  switch=( $(/sbin/ip route list | awk '/^default / { sub(/.* dev /, ""); print $1 }') )
+  ${BRCTL} delif ${switch} $1
+  /sbin/ip link set $1 down
+else
+  switch=( $(/bin/netstat -rn | awk '/^0\.0\.0\.0/ { print $NF }') )
+  ${BRCTL} delif ${switch} $1
+  /sbin/ifconfig $1 down
+fi
diff --exclude-from=/home/dang/.scripts/diffrc -up -ruN kvm-79.orig/scripts/qemu-ifup kvm-79/scripts/qemu-ifup
--- kvm-79.orig/scripts/qemu-ifup	2008-11-12 06:48:01.000000000 -0500
+++ kvm-79/scripts/qemu-ifup	2008-12-10 11:04:16.000000000 -0500
@@ -1,5 +1,22 @@
 #!/bin/sh
 
-switch=$(/sbin/ip route list | awk '/^default / { print $NF }')
-/sbin/ifconfig $1 0.0.0.0 up
-/usr/sbin/brctl addif ${switch} $1
+if [ -x /sbin/brctl ]; then
+  BRCTL="/sbin/brctl"
+elif [ -x /usr/sbin/brctl ]; then
+  BRCTL="/usr/sbin/brctl"
+else
+  echo "no bridge utils installed"
+  exit 1
+fi
+
+if [ -x /sbin/ip ]; then
+  switch=( $(/sbin/ip route list | awk '/^default / { sub(/.* dev /, ""); print $1 }') )
+  /sbin/ip link set $1 up
+else
+  switch=( $(/bin/netstat -rn | awk '/^0\.0\.0\.0/ { print $NF }') )
+  /sbin/ifconfig $1 0.0.0.0 up
+fi
+
+[[ ${switch#} -ne "1" ]] && logger -t kvm "$0 found more than one bridge connecting $1 to ${switch}"
+${BRCTL} addif ${switch} $1
+
